lib/sign_in/logingov/well_known/service.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module SignIn
+  module WellKnown
+    class Service
+      def openid_data
+        {
+          issuer: Settings.logingov.oauth_url,
+          authorization_endpoint: "#{Settings.logingov.oauth_url}/oauth2/authorize",
+          token_endpoint: "#{Settings.logingov.oauth_url}/oauth2/token",
+          token_refresh_endpoint: "#{Settings.logingov.oauth_url}/oauth2/refresh",
+          introspection_endpoint: "#{Settings.logingov.oauth_url}/oauth2/introspect",
+          end_session_endpoint: "#{Settings.logingov.oauth_url}/oauth2/logout",
+          token_revocation_individual_endpoint: "#{Settings.logingov.oauth_url}/oauth2/revoke",
+          revocation_endpoint: "#{Settings.logingov.oauth_url}/oauth2/revoke_all",
```

Comment:
These URLs aren't correct. These are login.gov's URLs. The need to be our URLs (env dependent)